From 4069463cd0caf027a0b01bfc46ebc30f310d6d61 Mon Sep 17 00:00:00 2001
From: Ding Wei <leo.ding@rock-chips.com>
Date: Wed, 17 Jun 2020 11:21:56 +0800
Subject: [PATCH] iommu: Export iommu_group_default_domain()

ERROR: "iommu_group_default_domain"
[drivers/video/rockchip/mpp/rk_vcodec.ko] undefined!

Change-Id: Ic6dc4a8c182f58963011bc9c4f7cbcddacdf315a
Signed-off-by: Ding Wei <leo.ding@rock-chips.com>
---
 drivers/iommu/iommu.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/iommu/iommu.c b/drivers/iommu/iommu.c
index 88cad7db1bf2..b463b2ab0580 100644
--- a/drivers/iommu/iommu.c
+++ b/drivers/iommu/iommu.c
@@ -1090,6 +1090,7 @@ struct iommu_domain *iommu_group_default_domain(struct iommu_group *group)
 {
 	return group->default_domain;
 }
+EXPORT_SYMBOL_GPL(iommu_group_default_domain);
 
 static int add_iommu_group(struct device *dev, void *data)
 {
-- 
2.35.3

